### DISCUSSION TITLE
[3097560] Liar's Bar {本体}

### DISCUSSION THREAD

**Discussion by XLpop** [pjy612/SteamManifestCache#882] [Share]
### Steam 游戏ID

`appid:3097560`

key:
```
3097561;86eb48bd45b0ad6b6c525996f1844bb03ddd150dff04f65f5119418b10017dd3
```

[3097561_6044670001040039826.zip](https://github.com/user-attachments/files/17484299/3097561_6044670001040039826.zip)

### 备注

在使用现有清单时，下载出现了配置错误。
不知是不是现有配置文件是否过时，在此更新一下。

email:Kminep@outlook.com

**Comment by pjy612** [comment]
* 分析格式错误
* #26

**Reply by XLpop** [reply]
已更新。
